api/src/api/admin.js

```diff
@@ -6,6 +6,13 @@ const Resource = require('../models/resource');
 const HomePage = require('../models/homepage');
 const Translation = require('../models/translation');
 
+const languageTypes = {
+  es: 'Spanish',
+  fr: 'French',
+  zh: 'Chinese',
+};
+const apiKey = '';
```

Comment:
Lets put this variable inside the `translateAndSaveText` to be more clear on what it's for! We can also set it equal to something like `process.env.GOOGLE_KEY` - we'll set `GOOGLE_KEY` in our .env files :)